TRANSLATING.md

```diff
@@ -108,6 +108,40 @@ If you spot something on [docs.astro.build](https://docs.astro.build/) that you
 
 Please see [CONTRIBUTING.md](https://github.com/withastro/docs/blob/main/CONTRIBUTING.md) for information about contributing via a fork, our Style Guide, and more!
 
+## Pull Request Recommendations
+
+To ensure translation contributions are sustainable, we have a few recommendations on how PRs should look like.
+
+### Recommended size
+
+PRs should try to be small, changing only a few pages/files and not surpassing the 300 lines changed mark. This is not a hard rule, there will be cases in which you will inevitably surpass the recommended limit, especially when the page you're translating is more than 500 lines long or when you need to change multiple pages to pass the CI checks.
+
+For smaller updates, especially those that only change a sentence or a code sample, you're completely free to bundle a dozen or so pages together, just be aware that the original pages may have other changes added to them which you will need to take care of, so the Translation Status Overview doesn't mistakenly think the page is 100% updated when your PR is merged.
+
+Following all this ensures PRs are feasible for reviewers to jump in, makes it easier for you to look if anything is outdated when some of the original pages change in the meantime, and reduce the risk of complex merge conflicts when multiple translators are working together.
```

Comment:
```suggestion
PRs should try to be small, changing only a few pages/files and not surpassing the 300 lines changed mark. This is not a hard rule! There will be cases in which you will change more than the recommended limit, especially when the page you're translating is more than 500 lines long or when you need to change multiple pages to pass the CI checks.

For smaller updates, especially those that only change a sentence or a code sample, you are welcome to bundle a small number (usually fewer than 10) pages together. Please be aware that the more pages you add, the longer it takes to get reviews! While your PRs are waiting, the original pages may already have other changes added to them. So it is important to keep checking your PRs to the current version of the English site. Your PR cannot be merged until it matches the English version, because the Translation Status Overview would mistakenly think the page is 100% updated when your PR is merged.

Following these guidelines ensures that PRs are small enough for reviewers to jump in and review quickly. It makes it easier for you to keep up with any changes to the original pages, and reduces the risk of complex merge conflicts when multiple translators are working together.
```